docs/index.adoc

```diff
@@ -100,7 +105,7 @@ class CheckSum implements Callable<Integer> {
     public Integer call() throws Exception { // your business logic goes here...
         byte[] fileContents = Files.readAllBytes(file.toPath());
         byte[] digest = MessageDigest.getInstance(algorithm).digest(fileContents);
-        System.out.printf("%0" + (digest.length*2) + "x%n", new BigInteger(1, digest));
+        System.out.printf(algorithm + " hash of " + file.getPath() + ": %0" + (digest.length*2) + "x%n", new BigInteger(1, digest));
```

Comment:
I agree that this makes the program a better `checksum` application. But I prefer the simplicity of the current version.

There are other ways in which we could improve the example, like allowing it to deal with multiple files, etc. But this would distract from the purpose, which is to give a glimpse of the power and ease of use of the library.

Reply:
Reverted back to original version.
I left the extended `printf` statement in the Jdoodle version since I think it makes sense here. This comes at the price that code sample in the manual and jdoodle are slight different. I think this is mor than outweighed by the fact that the user can immediately get a grasp what the application is doing when executing the jdoodle app.